### PR TITLE
THKS 51: Fix socket connection

### PR DESCRIPTION
**Pull request by mhrechkin**

### CODE REVIEW

File: client/.env.example [hunk 1]
```diff
@@ -5,5 +5,6 @@
 VITE_API_SERVER=http://localhost:3001
 VITE_SOCKET_SERVER=http://localhost:3002
```
Comment (v0ldemar01) [inline client/.env.example]: Please remove this variable, we don't use it anymore

File: client/vite.config.js [hunk 2]
```diff
@@ -24,7 +24,11 @@ const config = ({ mode }) => {
       host: VITE_HOST,
       port: Number(VITE_PORT),
       proxy: {
-        [VITE_API_PATH]: VITE_API_SERVER
+        [VITE_API_PATH]: VITE_API_SERVER,
+        [VITE_SOCKET_PATH]: {
```
Comment (v0ldemar01) [inline client/vite.config.js:28]: Did you have a chance to test it? I have some difficulties with it on my project and the config has the next view, but if your configuration is suitable, then will stay with it
image

**Review by v0ldemar01 — CHANGES_REQUESTED** [review]

File: client/vite.config.js (hunk 2 above)
Comment (mhrechkin) [reply client/vite.config.js:28]: Yes, i've tested it and it works correct. 
But i've encountered an issue with leaving reaction on post - toast message doesn't appear. I will create a separate task for this.

File: client/.env.example (hunk 1 above)
Comment (mhrechkin) [reply client/.env.example]: Now it is used, i forgot to change it's value in .env.example)

**Pull request merged**